### ISSUE TITLE
Strange message is output during downloading

### ISSUE THREAD

**Issue by AkiyukiOkayasu** [Ovyerus/bandsnatch#4]
Thank you for making great software. This seems to be working for me.  

However, a strange message appeared, so I thought I would report it here.  
Cookie.json was exported from Cookie Quick Manager.  
Here is the command I used. I ran it on macOS 13.2.  

```bash
bandsnatch -c cookies.json -f flac MY_NAME
```

```bash
thread 'tokio-runtime-worker' panicked at 'called `Result::unwrap()` on an `Err` value: Error("missing field `download_type`", line: 1, column: 3752)', src/api/mod.rs:188:91
stack backtrace:
   0: _rust_begin_unwind
   1: core::panicking::panic_fmt
   2: core::result::unwrap_failed
   3: bandsnatch::main::{{closure}}::{{closure}}::{{closure}}
   4: tokio::runtime::task::harness::Harness<T,S>::poll
   5: std::thread::local::LocalKey<T>::with
   6: tokio::runtime::scheduler::multi_thread::worker::Context::run_task
   7: tokio::runtime::scheduler::multi_thread::worker::Context::run
   8: tokio::macros::scoped_tls::ScopedKey<T>::set
   9: tokio::runtime::scheduler::multi_thread::worker::run
  10: <tokio::runtime::blocking::task::BlockingTask<T> as core::future::future::Future>::poll
  11: tokio::runtime::task::harness::Harness<T,S>::poll
  12: tokio::runtime::blocking::pool::Inner::run
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

I get this message, but the download seems to be working correctly.  
Therefore, there may be no harm done, but I reported it just in case.  

Again, your software is very useful. Thank you for making it.

**Comment by Ovyerus** [comment]
Looks like one of the releases it tries to download is missing a field for some reason, which results in one of the threads crashing (the others keep working so it downloads the rest as expected).

**Comment by AkiyukiOkayasu** [comment]
You are right some albums were not downloaded.  
I ran the same command several times and almost all the albums were downloaded.  
There were some albums that could not be downloaded after many times, but most were downloaded after a few runs.

**Comment by Ovyerus** [comment]
WIll be trying to fix this today/tomorrow

**Comment by Ovyerus** [comment]
Should be fixed in v0.2.0. Mind that the command to run has been changed from just `bandsnatch` to `bandsnatch run`, but same arguments as before.
